components/admin_console/feature_flags.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
+
+type Props = {
+    config: Record<string, unknown>;
```

Comment:
I think this type can be `AdminConfig` from `mattermost-redux/types/config`

Reply:
Awww man. Now I have to go make a redux PR :(

Reply:
https://github.com/mattermost/mattermost-redux/pull/1275

Reply:
Merged and ready to be updated to `f3ca7d9905ba9050a2a97f99fb78a002167f6fa4`

Reply:
Updated.